src/okta/services/okta.service.ts

```diff
@@ -91,14 +93,26 @@ export class OktaAuthService extends OktaAuth {
       });
       if (!this.token.isLoginRedirect()) {
         // Trigger an initial change event to make sure authState is latest
-        this.authStateManager.updateAuthState();
+        // Also starts the token auto-renew service
+        this.start();
       }
     }
 
+    ngOnDestroy(): void {
+      this.stop();
+    }
+
     private async emitAuthenticationState(state: boolean) {
       this.observers.forEach(observer => observer.next(state));
     }
 
+    public async isAuthenticated(): Promise<boolean> {
```

Comment:
👍 
before this change the config option is not used in code?

Reply:
let's also add a test for it

Reply:
There is already test for this: https://github.com/okta/okta-angular/blob/master/test/spec/service.test.ts#L150
It was failing after upgrade to okta-auth-js 5.
Reason - the implementation of `OktaAuth.isAuthenticated()` has been changed:
https://github.com/okta/okta-auth-js/blob/4.8/lib/browser/browser.ts#L443
https://github.com/okta/okta-auth-js/blob/5.0/lib/OktaAuth.ts#L484